### ISSUE TITLE
[form-core] set aria-live=assertive when validation feedback is shown on blur

### ISSUE THREAD

**Issue by tlouisse** [ing-bank/lion#1161]
## Expected behavior

A screen reader announces the error messages when it leaves the field (in case the message is shown on blur)
See: https://www.w3.org/WAI/tutorials/forms/notifications/#on-focus-change

## Actual Behavior

Currently, a the validation-feedback node gets `aria-live=polite`. This is fine when validation feedback is shown on keyup/modelValue change (error will be read right away, unless something more important was already being read).
On blur however, it leads to the focus shifting to the next input, causing the error message being read after the next focused element is being pronounced (this is confusing and not right in terms of priority).

The aria-live attribute should therefore be coupled to the current interaction state: when error will be shown after next blur, aria-live should be assertive, otherwise polite

**Comment by erikkroes** [comment]
I think we had some issues with assertive being less predictable than polite. It would be nice to test this by comparing the two situations.

**Comment by tlouisse** [comment]
I remember in the past that we needed to rerender content to make all screen readers understand (I believe it was VoiceOver + Safari and aria-live=polite). 
There must be a way to make this work
